docs(filter):initial FilterField ValueTemplate docs and Filter Refres… (#1971)
telerik/blazor-docs: components/filter/overview.md

| `Class` | `string` | The class that will be rendered on the outermost element. |
| `Value` | `CompositeFilterDescriptor` | Sets the value of the filter component. |

## Reference and Methods

> The Filter methods are accessible through its reference.

Reviewer: ```suggestion
The Filter exposes methods for programmatic operation. To use them, define a reference to the component instance with the `@ref` directive attribute.
```

> | Method | Description | Use-case
> | --- | --- | --- |
> | `Refresh` | Redraws the component. | Update the component when programmatically changing it during runtime. |

Reviewer: ```suggestion
| Method | Description |
| --- | --- |
| `Refresh` | Redraws the component. Use this method to update the component UI after changing its `Value` programmatically. |
```

                new FilterDescriptor
                {
                    Member = "Name",
                    MemberType = typeof(string),
                    Operator = FilterOperator.IsEqualTo
                },
                new FilterDescriptor
                {
                    Member = "AgeInYears",
                    MemberType = typeof(int),
                    Operator = FilterOperator.IsEqualTo
                }
            };
        await Task.Delay(1);
        FilterRef?.Refresh();
        ProcessGridData(FilterValue);
    }

    protected override void OnInitialized()
    {
        LoadData();
        base.OnInitialized();
    }

    private void LoadData()
    {
        for (int i = 1; i <= 30; i++)
        {
            InitialData.Add(new Person
                {
                    EmployeeId = i,
                    Name = "Name" + i,
                    AgeInYears = i + 20
                });
        }
        ProcessGridData(FilterValue);
    }
    
    public class Person
    {
        public int EmployeeId { get; set; }

        public string Name { get; set; } = string.Empty;

        public int AgeInYears { get; set; }
    }
}
````

> ## Events

Reviewer: Move the Events section before Parameter.
